pkg/storage/tree/flamebearer.go

```diff
@@ -75,7 +79,12 @@ func (t *Tree) FlamebearerStruct(maxNodes int) *Flamebearer {
 
 			xOffset += int(tn.Self)
 			otherTotal := uint64(0)
+			var otherNode *treeNode
 			for _, n := range tn.ChildrenNodes {
+				if bytes.Equal(n.Name, lostDuringRenderingName) {
+					otherTotal += n.Total
+					continue
+				}
```

Comment:
it's possible that there's already an "other" node from serialization. In this case we're storing the value and combine it with the values lost during rendering